examples/all-clusters-app/linux/fuzzing-main-pw.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <pw_fuzzer/fuzztest.h>
+#include <pw_unit_test/framework.h>
+
+#include "AppMain.h"
+#include <app/server/Server.h>
+
+#include <CommissionableInit.h>
+
+using namespace chip;
+using namespace chip::DeviceLayer;
+
+using namespace fuzztest;
+using namespace std;
+
+namespace {
+
+LinuxCommissionableDataProvider gCommissionableDataProvider;
+
+}
+
+// TODO #35369: move this to a test fixture once Test Fixture issues within FuzzTest are resolved.
+void CleanShutdown()
+{
+    Server::GetInstance().Shutdown();
+    PlatformMgr().Shutdown();
+    // TODO: We don't Platform::MemoryShutdown because ~CASESessionManager calls
+    // Dnssd::ResolverProxy::Shutdown, which starts doing Platform::Delete.
+    // Platform::MemoryShutdown();
+}
+
+void AllClustersFuzzer(const std::vector<std::uint8_t> & Bytes)
```

Comment:
I wonder whether it would be better to have some shared "take a bytespan and do stuff" thing that this and the existing fuzzer can share instead of just copy/pasting everything....